.github/workflows/ci.yml

```diff
@@ -1,6 +1,12 @@
 name: ci
 
-on: [push, pull_request]
+on:
+  push:
+    branches:
+      - master
+  pull_request:
```

Comment:
This means people cannot target other branches in pull requests. I have been doing this recently as I have many branches that stack upon each other and this has made it easier to chunk up large features into several smaller PRs. Is there a way around this limitation?

Reply:
I wonder if something like this is what we want:
```yaml
push:
    branches:
      - master
  pull_request:
```

Reply:
Yea, this sounds good.